src/unix/aix-ibmi.c

```diff
@@ -0,0 +1,292 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
New files should use `Copyright libuv project contributors. All rights reserved.`

Reply:
Updated the copyright text